package.json

```diff
@@ -39,6 +39,7 @@
         "pino-pretty": "^10.0.0",
         "rome": "12.1.3",
         "ts-node": "^10.9.1",
+        "tsconfig-paths": "^4.2.0",
```

Comment:
Whats this one for?

Reply:
I installed this to run the example files, but I removed the "example" script when I resolved conflicts in the package.json, I'll put it back

Reply:
Can it be removed now?